pkg/controller/jobs/statefulset/statefulset_webhook_test.go

```diff
@@ -249,76 +249,39 @@ func TestValidateUpdate(t *testing.T) {
 			wantErr: nil,
 		},
 		"change in queue label": {
```

Comment:
Add a test case for setting the queue label for the first time.

Reply:
Added on 972193d.

Reply:
lgtm